R/plot.fossils.R

```diff
@@ -107,6 +108,10 @@ plot.fossils = function(x, tree, show.fossils = TRUE, show.tree = TRUE, show.ran
   adj = NULL
   srt = 0
 
+  if (is.null(t.axis.lab)){
+    t.axis.lab = "Time before present"
```

Comment:
why not make "Time before present" the default instead of having NULL as default and then changing it ?